contracts/FlowFees.cdc

```diff
@@ -122,9 +121,15 @@ pub contract FlowFees {
         let tokenVault = payerAcct.borrow<&FlowToken.Vault>(from: /storage/flowTokenVault)
             ?? panic("Unable to borrow reference to the default token vault")
 
```

Comment:
Shouldn't this be deleted since you do it below?

Reply:
good catch! Forgot to delete it after changing it.